File: 1-adhock/1-bilangan-terbesar-tiga/solution.js
function bilanganTerbesarTiga(a, b, c) {
  maks = a;
  if (b > maks) {
    maks = b;
  }
  if (c > maks) {
    maks = c;
  }
  return maks;
}

console.log(bilanganTerbesarTiga(1, 2, 3)) // 3
console.log(bilanganTerbesarTiga(10, 30, 20)) // 30
console.log(bilanganTerbesarTiga(17, 15, 17)) // 17
console.log(bilanganTerbesarTiga(79, 50, 20)) // 79
console.log(bilanganTerbesarTiga(200, 300, 400)) // 400

File: 1-adhock/1-bilangan-terbesar-tiga/solution.go
package main

import "fmt"

func bilanganTerbesarTiga(a int, b int, c int) int {
	maks := a
	if b > maks {
		maks = b
	}
	if c > maks {
		maks = c
	}
	return maks
}

func main() {
	fmt.Println(bilanganTerbesarTiga(1, 2, 3))       // 3
	fmt.Println(bilanganTerbesarTiga(10, 30, 20))    // 30
	fmt.Println(bilanganTerbesarTiga(17, 15, 17))    // 17
	fmt.Println(bilanganTerbesarTiga(79, 50, 20))    // 79
	fmt.Println(bilanganTerbesarTiga(200, 300, 400)) // 400
}
